### DISCUSSION TITLE
Please allow "bad words"

### DISCUSSION THREAD

**Discussion by retrievaaaa** [community/community#108909] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

If i want to use fuck in my code copilot shouldn't judge me by refusing to work. You are artificially worsening your product for "political correctness".

**Comment by allancorrigall** [comment]
my queries regarding cannon.js physics bodies kept getting censored because of "bodies". It's a really unsophisticated filter on a sophisticated piece of software and its incredibly frustrating.
![gatekeeping progress](https://github.com/community/community/assets/6101356/a7c0e202-5b03-4580-b4f1-64d8acedf4e8)
